### ISSUE TITLE
Sharing collections of Albums

### ISSUE THREAD

**Issue by ScuttleSE** [photoview/photoview#414]
I suggest adding another hierarchy above albums; Collections.

They act just like an album, except that they collect Albums, instead of pictures.

Shareable in the same way as an album.

**Comment by viktorstrate** [comment]
I think this would be a nice addition and something similar has also been mentioned before here #322.

I have never used that feature of Picasa, but I'd like to hear a detailed suggestion as to how this might work in practice.

When you add an album to a collection are the sub-albums also added?
How might the collections be structured? should it be flat or hierarchical, ie. collections within collections.

**Comment by ScuttleSE** [comment]
So...I took some time to dig up an old copy of Picasa, took a few screenshots to give you an idea.

https://i.imgur.com/edw6uQF.png
This is Picasa after importing folders. Nothing at all fancy, just folders imported as Albums in a row. Much like Photoview now, but without the folder structure.


What you can do though is to move a folder to a collection. It doesn't move the actual folder, just virtually moves that folder in Picasa.

https://i.imgur.com/1Fx75rH.png

Here you can create subfolders to your hearts content. Files added to any of the folders/Albums in a collection will automatically show up in that album

https://i.imgur.com/hWEiz75.png

You can also create virtual Albums which is just a collection of pictures from other albums.

Basically, Picasa decouples the actual folder-structure from the structure inside Picasa. 

My suggestion would be something like this...

* Keep the version of "Albums" like it is today, or a version of it; a way to browse the actual filesystem.
* Add a version of the Collections where you can add an album and create a separate, virtual, structure.
* Add a way to create a "virtual" Album, where one can add pictures from one or several actual Albums
* The vitual Albums should act as "normal" ones, shareable the same way, addable to Collections and so on.
* You could even be real clever and create auto-updating virtual albums. "Add all pictures with the EXIF Tag 'Boats'" and stuff like that.

**Comment by viktorstrate** [comment]
Thank you for taking your time to dig out old Picasa and explaining it to me.

I really like the idea of splitting it up into Collections that group Albums and Virtual Albums that act exactly like "normal" albums but the user manually adds images to it.
